.rubocop.yml

```diff
@@ -26,6 +26,7 @@ Metrics/AbcSize:
 Metrics/BlockLength:
   Exclude:
   - 'config/routes.rb'
+  - 'app/contracts/new_representation_order_contract.rb'
```

Comment:
Is there a specific reason for this file to be excluded?

Reply:
Added due to the size of the Rep order payload.
However, I'm happy to move it to the inline syntax (`# rubocop:disable/enable`) if that's what you mean.

Reply:
Talked via Slack about this. Outcome: I missed the specific Cop that we're ignoring.